src/lib.rs

```diff
@@ -9,8 +9,15 @@
 pub mod bson;
 /// Module that contains the database types and constants.
 pub mod db;
+#[cfg(feature = "stardust")]
```

Comment:
I don't think we need to put `inx` behind a feature flag, or do you anticipate a use case for running the node without an INX interface?

Reply:
Could be that users only want to run the API over existing data, but I dunno.